### PR TITLE
Show filter status

### PR DESCRIPTION
**Pull request by karynas**

### CODE REVIEW

File: front-end/src/components/Admin/Grid/Filter/index.js [hunk 1]
```diff
@@ -46,9 +48,11 @@ class Filter extends Component {
     return fields;
   };
 
+  applyFilter = () => this.apply_filter.classList.add('filter__apply-btn-active');
```
Comment (Thealoner) [inline front-end/src/components/Admin/Grid/Filter/index.js]: This should be done in react-way. There is no need to use refs for this task.
You need to think of the 'enabled filter' as a state of the Filter component: add boolean property 'filtered' to the component state and insert the right class in render() depending on it.

File: front-end/src/components/Admin/Grid/Filter/index.js [hunk 2]
```diff
@@ -5,68 +5,72 @@ let defaultFilter = {
...
-    
+    const {columns} = this.props;
+
     let fields = columns.map(column => (
       <option key={column.field} value={column.field}>{column.title}</option>
     ));
 
-    fields.unshift(<option key="search" value="search">Все</option>);
-      
+    fields.unshift(<option key='search' value='search'>Все</option>);
```
Comment (Thealoner) [inline front-end/src/components/Admin/Grid/Filter/index.js]: Double quotes are used in JSX attributes by convention.

File: front-end/src/components/Admin/Grid/Filter/index.js [hunk 3]
```diff
@@ -5,68 +5,72 @@ let defaultFilter = {
...
-
         <span>
           <label>Значение: </label>
-          <input name="value" type="text" placeholder="" value={this.state.value} onChange={this.handleInputChange} />
+          <input name='value' type='text' placeholder='' value={this.state.value} onChange={this.handleInputChange}/>
         </span>
-
-        <button name="filter" type="submit" className="filter__apply-btn" ref={el => this.apply_filter = el} onClick={() => this.applyFilter()}>Применить фильтр</button>
-        <button name="clear" className="filter__clear-btn" onClick={this.clearFilter} type="button">Очистить фильтр</button>
+        <button name='filter' type='submit' disabled={this.state.filtered}>
```
Comment (Thealoner) [inline front-end/src/components/Admin/Grid/Filter/index.js]: Double quotes are used in JSX by convention.

File: front-end/src/components/Admin/Grid/Filter/index.js (hunk 2 above)
Comment (YaroslavNikiforov) [reply front-end/src/components/Admin/Grid/Filter/index.js]: where I can read this convention ?

**Pull request merged**